ci/request_pr_exp.py

```diff
@@ -111,6 +121,11 @@ def _parse_args(cmd) -> argparse.Namespace:
                       type=str,
                       default=LLM_NAME,
                       help=f'Large Language Model name, default: {LLM_NAME}.')
+  parser.add_argument(
+      '--llm-locations',
```

Comment:
nit: an abbrev. flag? E.g., `-ll`.
This will minimize our working when using the command.